drake/solvers/Optimization.h

```diff
@@ -597,6 +598,10 @@ class DRAKEOPTIMIZATION_EXPORT OptimizationProblem {
       getLinearEqualityConstraints() const {
     return linear_equality_constraints;
   }
+  const std::list<Binding<LinearConstraint>>&
```

Comment:
What is the policy here?  Are we enforcing that any change to a class with non-compilant function names requires a full class cleanup?  Or that new functions be compliant?